diagnostic.md

```diff
@@ -177,5 +207,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+The screen width will be at 25%
```

Comment:
The _element_ with be 137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px.